docs/maintainers/growing-your-project.md

```diff
@@ -18,33 +18,36 @@ When it comes to deciding on whether your open source project is ready to grow,
 - Are there tasks that are becoming too overwhelming for me and/or team?
 - How can I improve and maintaining my project's current and pending features?
 - How many stars and forks do I want to reach this time, and who can help me achieve it?
-If you have said yes to these questions, it's time to develop your growth strategies. If you haven't already, consider creating different teams for specific purposes. For example, you can create a docs team for documentation maintenance, a triage team for reviewing and improving issues, or a maintainer team to help you with overall oversight of the project.  Once you have decided the types of teams you want to add to your project, you can create separate pages for each team in your project's Workspace page. Check out the [Creating a New Repository Insights Page](../docs/maintainers-guide.md#adding-repositories-to-your-workspace).
+If you have said yes to these questions, it's time to develop your growth strategies. If you haven't already, consider creating different teams for specific purposes. For example, you can create a docs team for documentation maintenance, a triage team for reviewing and approving issues, or a maintainer team to help you with overall oversight of the project.  Once you have decided the types of teams you want to add to your project, you can create separate pages for each team in your project's Workspace page. Check out the [Creating a New Repository Insights Page](../docs/maintainers-guide.md#adding-repositories-to-your-workspace).
 
 ### How to identify new contributors and team members
 
-Like Superman, Wonder Woman, the Flash, and Batman in ***The Justice League***, the people you bring on your team(s) and your project's contributors are key to its success, which is why picking the right people is important. When doing this process, consider going to their OpenSauced profiles and ask yourself the following questions:
+Like Superman, Wonder Woman, the Flash, and Batman in ***The Justice League***, the people you bring on your team(s) and your project's contributors are key to its success, which is why picking the right people is important.  When doing this process, consider going to their OpenSauced profiles and checking their [dev card](../docs/features/dev-card.md). While you do this, ask yourself the following questions:
 
 - How often do they contribute to yours and other open source projects in general?
 - Are they invested in your project's success?
 
```

Comment:
We have to be specific as to whether we want them to create a repository insight page or a contributor insight page.

Reply:
Hey @BekahHW. Here, I decided that maintainers could use a contributor insight page. I got the inspiration from [this part of the maintainers guide](https://docs.opensauced.pizza/maintainers/maintainers-guide-to-open-sauced/#why-create-a-contributor-insight-page).